seeds/seed.js

```diff
@@ -0,0 +1,25 @@
+const sequelize = require('../config/connection');
+const { User, Favorite, Movie } = require('../models');
+
+// const userData = require('./userData.json');
+// const projectData = require('./projectData.json');
+
+const seedDatabase = async () => {
+  await sequelize.sync({ force: true });
+
```

Comment:
We aren't currently seeding anything in this function. I realize it's probably because we were trying to figure out the Models, but figured I'd mention it 🤷‍♂️

Reply:
it creates the tables

Reply:
i think.  i know there's no actual seed data, but there will be someday